src/main/java/org/broadinstitute/hellbender/tools/LeftAlignIndels.java

```diff
@@ -35,7 +35,7 @@
  *
  * <h3>Usage example</h3>
  * <pre>
- * ./gatk-launch LeftAlignIndels \
+ * ./gatk LeftAlignIndels \
```

Comment:
@sooheelee wants all instances of `./gatk` in the docs changes to `gatk`

Reply:
done

Reply:
I left it as ./ in the readme because I think people will copy and paste and be confused.